draft-ietf-ppm-dap.md

```diff
@@ -1214,52 +1200,48 @@ aggregation ({{aggregate-flow}}) and finalizing the aggregate shares.
 
 ## Task Configuration {#task-configuration}
 
-Prior to the start of execution of the protocol, each participant must agree on
-the configuration for each task. A task is uniquely identified by its task ID:
+A task represents a single measurement process, though potentially aggregating
+multiple, non-overlapping batches of measurements. Each participant in a task
+must agree on its configuration prior to its execution. This document does not
+specify a mechanism for distributing task parameters among participants.
+
+A task is uniquely identified by its task ID:
 
 ~~~ tls-presentation
 opaque TaskID[32];
 ~~~
 
-The task ID value MUST be a globally unique sequence of bytes. Each task has
-the following parameters associated with it:
-
-* `leader_aggregator_url`: A URL relative to which the Leader's API resources
-   can be found.
-* `helper_aggregator_url`: A URL relative to which the Helper's API resources
-  can be found.
```

Comment:
GitHub's diff strategy is really unfortunate here. I want to highlight that the leader and helper URLs are still task parameters, but I removed the declarations `leader_aggregator_url` and `helper_aggregator_url` since they aren't referenced anywhere, so we don't need this bit of pseudocode pseudoformalism.